diff --git a/prepare_vscode.sh b/prepare_vscode.sh
index 511f9b3..fa52ab9 100755
--- a/prepare_vscode.sh
+++ b/prepare_vscode.sh
@@ -143,7 +143,7 @@ setpath "product" "tipsAndTricksUrl" "https://go.microsoft.com/fwlink/?linkid=85
 setpath "product" "twitterUrl" "https://go.microsoft.com/fwlink/?LinkID=533687"
 
 if [[ "${DISABLE_UPDATE}" != "yes" ]]; then
-  setpath "product" "updateUrl" "https://codex.now.sh"
+  setpath "product" "updateUrl" "https://vscodium-update-api-steel.vercel.app"
   setpath "product" "downloadUrl" "https://github.com/BiblioNexus-Foundation/codex/releases"
 fi
 
